core/domain/feedback_services.py

```diff
@@ -500,6 +502,9 @@ def get_thread_summaries(user_id, full_thread_ids):
     thread_summaries = []
     number_of_unread_threads = 0
     for index, thread in enumerate(threads):
+        if not feedback_thread_user_models[index]:
+            update_messages_read_by_the_user(
```

Comment:
This will only be called if the feedback_thread_user_model is not present. After a time we would not expect this to be called very much. And at the start, there wouldn't be much data for this to make much of a difference. And plus this just creates a model. It doesn't really update the messages.